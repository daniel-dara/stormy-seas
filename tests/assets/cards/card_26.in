###X###--
#A#x###--
#A##B##-C
-#-#B###C
E##-#-##H
E#-###-#H
-###-###H
####-##--