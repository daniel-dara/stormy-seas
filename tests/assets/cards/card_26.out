8R1, ED1, HD1, 5R1, BD1, BL1, AD1, 1R2, CL1, AU2, AL1, 1L1, XD1, HU5, 6R1, 7R1, BD1, BL1, CD1, CR1, 4R1, XD2, BU1, CD1, 4R1, ED1, 6L1, XD5